PlutoHelperAgent/ProjectLockerAndKeychainFunctions.m

```diff
@@ -108,7 +108,7 @@ + (NSDictionary *) load_data_from_keychain {
 + (NSURLSessionTask *) communicate_with_server:(NSString*)url :(NSString*)method :(NSString*)type :(NSDictionary*)body :(BOOL)send_cookie :(BOOL)test_connection
```

Comment:
is `(BOOL)test_connection` used any more? If not, please remove it

Reply:
No. Done.